_posts/2019-11-27-gettingstarted-2020/getting-started-2020.Rmd

```diff
@@ -0,0 +1,144 @@
+---
+title: "Getting started with Keras from R - the 2020 edition"
+description: > 
+ Looking for materials to get started with deep learning from R? This post presents useful tutorials, guides and background documentation on the new TensorFlow for R website. 
+ Advanced users will find pointers to applications of new release 2.0 (or upcoming 2.1!) features alluded to in the recent TensorFlow 2.0 post.
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydana2019gettingstarted2020
+date: 11-27-2019
+categories:
+  - Introductions
+  - Keras
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/website.png
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+
+If you've been thinking about diving into deep learning for a while -- using R, preferentially --, now is a good time. For TensorFlow / Keras, one of the predominant deep learning frameworks on the market, last year was a year of substantial changes; for users, this sometimes would mean ambiguity and confusion about the "right" (or: recommended) way to do things. By now, [TensorFlow 2.0](https://blogs.rstudio.com/tensorflow/posts/2019-10-08-tf2-whatchanges/) has been the current stable release for about two months; the mists have cleared away, and patterns have emerged enabling leaner, more modular code that accomplishes a lot in just a few lines.
+
+To give the new features the space they deserve, and assemble central contributions from related packages all in one place, we have significantly remodeled the [TensorFlow for R website](https://tensorflow.rstudio.com/). So this post really has two objectives. 
+
+First, it would like to do exactly what is suggested by the title: Point new users to resources that make for an effective start into the subject. 
+
+Second, it could be read as a "best of new website content". Thus, as an existing user, you might still be interested in giving it a quick skim, checking for pointers to new features that appear in familiar contexts. To make this easier, we'll add side notes to have new features stand out in the text.
```

Comment:
```suggestion
Second, it could be read as a "best of new website content". Thus, as an existing user, you might still be interested in giving it a quick skim, checking for pointers to new features that appear in familiar contexts. To make this easier, we'll add side notes to highlight new features.
```